src/styles/boxes/container.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  margin: 50px auto 0;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: mainWidth($distanceBetweenCards, $cardsWidth);
+  gap: $distanceBetweenCards;
```

Comment:
remove gap and add justify-content: space-between to add space between cards with flex